src/survey-enfr.json

```diff
@@ -1104,7 +1104,7 @@
                                 {
                                     "value": "item5",
                                     "text": {
-                                        "default": "Process optimization & workflow automation: Analyzing large data sets to identify and anomalies, cluster patterns, predict outcomes or ways to optimize; and automate specific workflows",
```

Comment:
The only ampersand in the entire file. Caused one problem (fixed through code), but using "and" seems safer and may prevent future issues.